src/Datadog.NativeLoader/class_factory.h

```diff
@@ -0,0 +1,28 @@
+#ifndef CLASS_FACTORY_H_
+#define CLASS_FACTORY_H_
+
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full
+// license information.
+
+#include <atomic>
+#include "unknwn.h"
+#include "proxy.h"
+
+class ClassFactory : public IClassFactory
+{
+private:
+    std::atomic<int> refCount;
+    datadog::nativeloader::DynamicDispatcher* dispatcher;
```

Comment:
style: how do we make the difference between member and non-member variables like `m_XXX`? (C# there the `_` added in front)

Reply:
adding `m_XXX` prefix...

Reply:
done in eb51a06